src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="ohrm_buzz_comment")
+ * @ORM\Entity
+ */
+class BuzzComment
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="bigint")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private int $id;
+
+    /**
+     * @var BuzzShare
+     *
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\BuzzShare")
+     * @ORM\JoinColumn(name="share_id", referencedColumnName="id")
+     */
+    private BuzzShare $share;
+
+    /**
+     * @var Employee
+     *
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\Employee")
+     * @ORM\JoinColumn(name="employee_number", referencedColumnName="emp_number")
+     */
+    private Employee $employee;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="number_of_likes", type="integer", nullable=true)
+     */
+    private int $numOfLikes = 0;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="comment_text",  type="string", nullable=true)
+     */
+    private string $text;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(name="comment_time", type="datetime")
```

Comment:
Seems text is required. If so, do we need to make this field nullable?

Reply:
no need to keep it nullable. I have revert it to make backward compatible with 4.x